src/sentry/utils/security/orgauthtoken_jwt.py

```diff
@@ -9,13 +9,15 @@
 
 
 def generate_token(org_slug: str, region_url: str):
+    sentry_url = settings.SENTRY_OPTIONS.get("system.url-prefix") or settings.SENTRY_URL_PREFIX
```

Comment:
cc @markstory at least in my dev environment, `settings.SENTRY_OPTIONS.get("system.url-prefix")` was `None`, so I added this fallback?

Reply:
Hmm, mypy complains:

```
src/sentry/utils/security/orgauthtoken_jwt.py:12: error: 'Settings' object has no attribute 'SENTRY_URL_PREFIX'  [misc]
```

but it _does_ seem to work in my dev env at least 😅

Reply:
hmm, actually, nevermind, now it seems to be working as expected... so removed this again! Not sure why it was complaining before 😓